lib/snippets.coffee

```diff
@@ -28,6 +28,14 @@ module.exports =
 
     snippets = this
 
+    # when snippets package settings changes, reload all package snippets
+    atom.config.onDidChange('snippets', =>
+      if @loaded
+        @unloadPackageSnippets()
+        @loadAll()
```

Comment:
This part worries me:

* We unload package snippets and then `loadAll`. `loadAll` also loads bundled snippets (The snippets included by the snippets pacakge) and user snippets both of which are never unloaded before loading them again.
* I don't think we need to unload all package snippets because one package got disabled or enabled. This needs to change to only affect that package that changed. I'm seeing a significant slowdown when toggling these settings on a slower computer.